modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -156,3 +156,23 @@ function stanford_profile_styles_preprocess_field__react_paragraph_row(&$variabl
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_block().
+ */
+function stanford_profile_styles_preprocess_block(&$variables) {
+
+  // Alter the styles and values for the config form blocks.
+  $plugin = $variables['plugin_id'];
+  if ($plugin == "config_pages_block") {
```

Comment:
you can include this in the hook. `hook_preprocess_block__config_pages_block` to make the hook more granular

Reply:
Yeah, good call.